Packs/AWS-GuardDuty/pack_metadata.json

```diff
@@ -11,7 +11,7 @@
         "Cloud Services"
     ],
     "tags": [
-        "Data Source"
+        "xsiam:Data Source"
```

Comment:
The prefix is `xsiam`? No `marketplacev2`?

Reply:
Yes, will change

Reply:
@tkatzir - I think it should be `xsiam` and not `marketplacev2` as we are planning in the future to not use marketplacev2 anymore and use xsiam instead.